get_data/coronadata.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+import pandas as pd
+import requests
+
+from get_data.get_data import call_api
+
+# Date from the day before the outbreak in italy
+outbreakDate = date(2020, 2, 21)
+today = date.today()
+days = (today - outbreakDate).days
+
+# Collect prices and normalise to see the overall effect on the stock
```

Comment:
This comment could go inside the function as docstring, ie """Collect prices and... """